output.md

```diff
@@ -90,59 +95,73 @@ file.h5
         ...
     snapshot_1/
         attribute: t
-        attribute: ts 
+        attribute: ts
         space_1/
         ...
         space_n/
     ...
     snapshot_n/
 ```
-In addition to that, psydac also has a class to read those files, recreate all the `FemSpace` and `FemField` objects and export them to `VTK`. 
```

Comment:
- Capitalize: psydac -> Psydac
- Please add class name: PostProcessManager

Reply:
Fixed in [34bfd98](https://github.com/pyccel/psydac/pull/234/commits/34bfd9845a7a7c44d8589ff9b670b919f204d457)